buildSrc/private/src/main/kotlin/androidx/build/clang/KonanBuildService.kt

```diff
@@ -233,9 +236,11 @@ abstract class KonanBuildService @Inject constructor(
                 it.parameters.konanHome.set(
                     nativeCompilerDownloader.compilerDirectory
                 )
-                it.parameters.prebuilts.set(
-                    project.getKonanPrebuiltsFolder()
-                )
+                if (!ProjectLayoutType.isPlayground(project)) {
+                    it.parameters.prebuilts.set(
+                        project.getKonanPrebuiltsFolder()
```

Comment:
Can we assert it is not null here? I would not want to accidentally start downloading these in CI.

Reply:
not sure if i understand.
this is where we set the parameters so there is nothing to assert.

I could try to assert when we create the distribution (where we read the parameter) but the build service doesn't have access to `Project` so I'm not sure if i can check for ` if(!playground) assertNotNullPrebuilts()`. one option is to pass the project layout type as an argument to the build service.

Is that what you prefer or am i misunderstanding this?

Reply:
implemented the option where i pass project layout type.

Reply:
where is this implementation? I cannot seem to find it.

Reply:
mb. the code push didn't go through 🤦  updated now.

Reply:
FYI this is the check: https://github.com/androidx/androidx/pull/647/files#diff-4d57a5bf411f5c52d9b975b078f74b4c8034a31922c196ad836485a03f1f130fR56